README.md

```diff
@@ -904,6 +912,7 @@ ONE TWO THREE
 If you want to capture the output of a process, you can provide a suitable `Eio.Flow.sink` as the `stdout` argument,
```

Comment:
I disabled the process tests.  There are two problems:

1. Process operations are not yet implemented on Windows.
2. Even if the process operations were implemented on Windows, the tests would not work on Windows, because they use commands that do not necessarily exist on Windows.

Reply:
Yeah. We really need a way to selectively disable tests on different platforms (e.g. https://github.com/realworldocaml/mdx/issues/393).